dubbo-common/src/main/java/org/apache/dubbo/common/URL.java

```diff
@@ -1058,6 +1039,19 @@ public URL addParameters(Map<String, String> parameters) {
         return new URL(protocol, username, password, host, port, path, map);
     }
 
+    /**
+     * test if parameters are already contained in the url
+     */
+    private boolean contains(Map<String, String> parameters) {

```

Comment:
instead of contains I think **isEqualParameter** would be better name from the task this method is doing. What do you say?